app/tydom/TydomClient.py

```diff
@@ -52,6 +54,56 @@ def __init__(
             self.cmd_prefix = ""
             self.ping_timeout = None
 
+    @staticmethod
+    def getTydomCredentials(login: str, password: str, macaddress: str):
+        """get tydom credentials from Delta Dore"""
+        try:
+            response = requests.get("https://deltadoreadb2ciot.b2clogin.com/deltadoreadb2ciot.onmicrosoft.com/v2.0/.well-known/openid-configuration?p=B2C_1_AccountProviderROPC_SignIn")
```

Comment:
Could the hard-coded `https://deltadoreadb2ciot.b2clogin.com` root url be externalized as a constant?
It should avoid duplication and simplify refactoring if the value changes in the future.